R/show.R

```diff
@@ -0,0 +1,15 @@
+#' Show `teal_data` object
+#'
+#' Prints `teal_data` object
+#' @param object (`teal_data`)
+#' @return nothing
```

Comment:
wrong

Reply:
Frankly it looks like it returns the input object lol. So we need to update this as well https://github.com/insightsengineering/teal.code/blob/main/R/qenv-show.R#L5

Reply:
05d385c34986afbe3c2a3c65818ce6c96806db4f

Reply:
> Frankly it looks like it returns the input object lol.

As a proper print method should.

> So we need to update this as well

done in insightsengineering/teal.code#166

Reply:
thanks. I blindly copied the content and didn't review.